src/Sun.svelte

```diff
@@ -33,7 +33,7 @@
 <svelte:window bind:innerWidth={width} bind:innerHeight={height} />
```

Comment:
instead of using window and innerWidth please use the following:

```
<div bind:offsetWidth={width} bind:offsetHeight={height}>
    <svg width={width} height={height}>
```